nestkernel/model_manager.h

```diff
@@ -331,26 +324,11 @@ ModelManager::assert_valid_syn_id( synindex syn_id, thread t ) const
   }
 }
 
-inline void
-ModelManager::delete_secondary_events_prototypes()
-{
-  for ( auto it = secondary_events_prototypes_.begin(); it != secondary_events_prototypes_.end(); ++it )
-  {
-    for ( std::map< synindex, SecondaryEvent* >::iterator iit = it->begin(); iit != it->end(); ++iit )
-    {
-      ( *iit->second ).reset_supported_syn_ids();
-      delete iit->second;
-    }
-  }
-  secondary_events_prototypes_.clear();
-}
-
 inline SecondaryEvent&
-ModelManager::get_secondary_event_prototype( const synindex syn_id, const thread tid ) const
+ModelManager::get_secondary_event_prototype( const synindex syn_id, const thread tid )
```

Comment:
Why is this no longer `const`? All the "get"ting suggests const-ness.

Reply:
I seem to remember that something down the line was not `const`. Not sure anymore, but there was a good reason.